src/main/java/org/folio/marccat/integration/TenantService.java

```diff
@@ -174,7 +174,7 @@
    * @throws SQLException the SQL exception
    * @throws IOException  Signals that an I/O exception has occurred.
```

Comment:
Also remove  * @throws IOException  Signals that an I/O exception has occurred.

Reply:
Done.